web/src/org/openmrs/module/chica/web/ManualCheckinSSNMRN.java

```diff
@@ -63,24 +63,23 @@ public static void verifyMRN(HttpServletRequest request, HttpServletResponse res
 		
 		EncounterService encounterService = Context.getEncounterService();
 		List<Encounter> encounters = encounterService.getEncountersByPatientIdentifier(mrn);
-		int encounterSize = encounters.size();
-				
+						
 		if (valid) {
 			pw.write(XML_RESULT_START + XML_TRUE + XML_RESULT_END);
 		} else {
 			pw.write(XML_RESULT_START + XML_FALSE + XML_RESULT_END);
 		}
 		
-		boolean sessionValid = false;
+		boolean validEncounter = false;
 
-		if (encounterSize != 0) {
-			sessionValid = true;
+		if (encounters.size() != 0) {
+			validEncounter = true;
 		}
 		
-		if (sessionValid) {
-			pw.write(XML_SESSION_RESULT_START + XML_TRUE + XML_SESSION_RESULT_END);
+		if (validEncounter) {

```

Comment:
Lines 73 - 77 don't seem to be needed if you just add if(encounters.size() > 0) here instead of checking if(validEncounter)